oat-sa/tao-test-runner-qti-fe
# [TAO-9727] WTP-1491 - ACT - End test warning msg displays incorrect no. of unanswered questions.

Related to : https://oat-sa.atlassian.net/browse/TAO-9727
 
Add possibility to calculate stats based on test scope, but with consideration that all inaccessible items answered to map helper. Extend message helper to be able to provide exit message based on this.
  
#### How to test
 
Describe testing process...
 - Prepare an instance of tao with taoAct extension
- Prepare with two tests with multiple parts. The first test should have linear navigation for all the parts. The second - no linear. Also parts should allow exit from test
- Try to exit from test somewhere in the middle
- Check exit messages for both tests
  
#### Dependencies
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1270
